core/storage/user/gae_models.py

```diff
@@ -3011,3 +3011,141 @@ def get_export_policy(cls) -> Dict[str, base_models.EXPORT_POLICY]:
         """
         empty_dict: Dict[str, base_models.EXPORT_POLICY] = {}
         return dict(super(cls, cls).get_export_policy(), **empty_dict)
+
+
+class LearnerGroupUserModel(base_models.BaseModel):
+    """Model for storing user's learner groups related data.
+
+    Instances of this class are keyed by the user id.
+    """
+
+    # List of learner group ids which the user has been invited to join.
+    invited_to_learner_groups_ids = (
+        datastore_services.StringProperty(repeated=True, indexed=True))
+    # List of learner group ids which the user has joined as a student.
+    student_of_learner_groups_ids = (
+        datastore_services.StringProperty(repeated=True, indexed=True))
+    # List of ProgressSharingPermission dicts, which is defined in
+    # user_domain.py, each dict corresponds to a learner group and
+    # has stores progress sharing permission status for that group.
+    progress_sharing_permissions_list = (
+        datastore_services.JsonProperty(repeated=True, indexed=False))
```

Comment:
I don't think we usually use `_list` ending of the properties in models. Should it be just `progress_sharing_permissions`?